TicketsDemo.Domain/DefaultImplementations/ReservationLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketsDemo.Domain.Interfaces;
+
+namespace TicketsDemo.Domain.DefaultImplementations
+{
+    public class ReservationLogger : ILogger
+    {
+        public string _logLocation;
+        public ReservationLogger()
+        {
+            _logLocation = AppDomain.CurrentDomain.BaseDirectory + "log.txt";
```

Comment:
it would be better to read path from separate ISettings service or smth like that. Where ISettings will return all configurable settings of your app, such as logging file path.

interface ISettings {
string LogFilePath {get;set;}
}